README.md

```diff
@@ -88,6 +88,18 @@ can deploy with default configuration file and then modify
 You may need to restart SR-IOV device plugin pods to catch up configuration
 file changes.
 
+The operator will also deploy a new network attachment definition for SR-IOV
+network. By default, its name is `sriov-network` but it can be changed using
+the `SRIOV_NETWORK_NAME` environment variable.
+
+One may also want to change the type of the newly created network attachment
+definition from the default `sriov` to something else. For example, this is
+needed for Red Hat's CNV product that is deployed on top of OpenShift that may
+be already shipped with `sriov` CNI plugin of different version incompatible
+with KubeVirt. In this case an admin may want to use different network types
+for KubeVirt and OpenShift SR-IOV CNI plugins. This can be achieved using the
+`SRIOV_NETWORK_TYPE` environment variable.
```

Comment:
Would it make sense to use this variable to deploy the binary? So we could simply build operator manifest with different environment variable while building for CNV?

Now it is done by combining adjusted sriov image build and build of operator. If we keep it in here, it would be easier even on u/s. i.e. use the same image with different binary name.

Reply:
This is a good idea.

Reply:
Interestingly, I didn't need to change a word in README for this.